devolving_music/views/submit_spotify_playlist.py

```diff
@@ -11,6 +11,9 @@
 class SubmitSpotifyPlaylistView(View):
     @safe_json_params
     def post(self, _request, playlist_link: str, event: Event):
+        if not event.allow_spotify:
+            return failure("Spotify submissions not permitted for this event.")
```

Comment:
Maybe it would be good to move "Spotify submissions not permitted for this event." into the event class as a static function called error(key) since this code is repeated a few times and we may have more platforms in the future. It would be nice to be able to modify all error messages resulting from event in one place.

Reply:
Not a bad idea.

Reply:
The reason I didn't is that there's no logical requirement that the error messages be exactly the same except for the platform name, since we're not trying to parse them or anything, but nonetheless I'm not opposed to the idea.

Reply:
What do you mean the submission views not understand which platform they pertain to? Isn't it clear by the name of the submission view?

Reply:
I added a `PLATFORM` class variable to both views, that we could use for an equality check or something in another circumstance.